GiniVision/Classes/Core/Screens/Screen API Coordinator/GiniScreenAPICoordinator.swift

```diff
@@ -208,6 +221,9 @@ extension GiniScreenAPICoordinator {
     }
     
     @objc func showHelpMenuScreen() {
+        
+        trackingDelegate?.onCameraScreenEvent(event: Event(type: .help))
```

Comment:
Documentation should mention, that `.help` is triggered only when using the Screen API.